plugins_src/commands/wpc_sel_win.erl

```diff
@@ -276,6 +276,14 @@ handle_cast({new_state, #{ssels:=New} = SS}, #state{lc=LC, ss=Old, shown=OS}=Sta
 	    ignore
     end,
     {noreply, State#state{lc=LC, shown=Shown, ss=SS}};
+handle_cast({ssels,deselect}, #state{lc=LC}=State) ->
+    Sel = wxListCtrl:getNextItem(LC, -1, [{state,?wxLIST_STATE_SELECTED}]),
+    if Sel >= 0 ->
+	%% remove any selection so the context menues can be properly processed
+	wxListCtrl:setItemState(LC, Sel, 0, ?wxLIST_STATE_SELECTED);
+    true -> ok
+    end,
+    {noreply, State#state{sel=none}};
```

Comment:
You should do this in this clause:
```
handle_cast({new_state, #{ssels:=New} = SS}, #state{lc=LC, ss=Old, shown=OS}=State) ->
    Shown = update_sels(New, Old, OS, LC),
    case SS of
	#{sh:=false, mode:=Mode} ->
	    try array:map(fun(Index, {M, _}) when M =:= Mode ->
				  throw({first, Index});
			     (_, M) -> M
			  end, Shown)
	    catch {first, Index} ->
		    wxListCtrl:ensureVisible(LC, Index)
	    end;
	_ ->
	    ignore
    end,
    {noreply, State#state{lc=LC, shown=Shown, ss=SS}};
```

And do it in a function reset_select(..) 
The event new_state is only sent when selection changed, see forward_event.
And if that is not good enough you should modify forward_event so that it fit your needs,
no modification outside of this module is needed for the event.